src/cider/nrepl/middleware/stacktrace.clj

```diff
@@ -104,7 +104,7 @@
   [{:keys [class] :as cause}]
   (if (= class "clojure.lang.Compiler$CompilerException")
     (update-in cause [:message] str/replace
-               #".* (compiling:)\((.*)\)" "Error $1 $2")
+               #".*?: (.*?), (compiling:)\((.*)\)" "Error $2 $3 $1")
```

Comment:
Nope, slightly different [1].  In this case, you would probably be safe with either variation, but I'm being paranoid.

```
user=> (clojure.string/replace-first "a:b c:d" #"(.*):(.*)" "$1x$2")
"a:b cxd"
user=> (clojure.string/replace-first "a:b c:d" #"(.*?):(.*)" "$1x$2")
"axb c:d"
```

[1] http://www.regular-expressions.info/repeat.html#lazy